torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -126,53 +143,78 @@ bool removeExtraNodeOutputs(Node* n) {
   return initial_outputs != true_block->outputs().size();
 }
 
-void ConstantPropagation(Block* block, const AliasDb& aliasDb, bool recurse);
+// remove extra outputs from the node
+void removeExtraLoopOutputs(Node* node) {
+  auto loop_body = node->blocks().at(0);
+  auto loop_input_offset = 2; // offset of loop carried deps in input list
```

Comment:
pool this and the other one into a `static const`

Reply:
I can change it - fwiw I was following the style of similar code that exists in dead_code_elimination & uses auto